eng/environments/dev.sh

```diff
@@ -1,11 +1,12 @@
 #!/bin/bash
+CLOUD_ENV="dev"
```

Comment:
No need for modifications in these env files, can use the ones from the `monorepo` to get the `CLOUD_ENV` values